Resources/Prototypes/_EE/Entities/Mobs/Player/ipc.yml

```diff
@@ -79,6 +79,7 @@
     entityType: enum.SiliconType.Player
     batteryPowered: true
     drainPerSecond: 1.3 # DeltaV change from 1.5
+    idleDrainReduction: 0.6 # TheDen - 60% less power usage while idling, for IPC Dynamic Power Draw
```

Comment:
On one hand, 0.6 is the default value so this is redundant. On the other it adds an explicit comment about something that was changed.
So I'm not sure if this line should be here or not

Reply:
I'd say leave it in case people know it's there tbh, up to you though, it's 0.9 on den so